distro/pkg/rpm/netopeer2.spec

```diff
@@ -65,13 +65,12 @@ a single established NETCONF session.
 %cmake -DCMAKE_BUILD_TYPE=RELWITHDEBINFO \
        -DSYSREPO_SETUP=OFF \
        -DPIDFILE_PREFIX=/run \
-       -DSERVER_DIR=%{_libdir}/netopeer2-server
+       -DSERVER_DIR=%{_var}/lib/netopeer2-server
```

Comment:
My RPM manual says that it's `%{_sharedstatedir}` for (an equivalent of) `/var/lib`.